contrib/verifybinaries/verify.py

```
@@ -0,0 +1,202 @@
+            offending_files.append(binary_filename)
+    if offending_files:
+        print("Hashes don't match.")
+        print("Offending files:")
+        print('\n'.join(offending_files))
+        return 1
+    verified_binaries = [entry[1] for entry in hashes_to_verify]
+
+    # clean up files if desired
+    if len(args) >= 2:
+        print("Clean up the binaries")
+        remove_files([sigfile1, sigfile2] + verified_binaries)
```

Ruff (0.15.7)

[warning] Consider `[sigfile1, sigfile2, *verified_binaries]` instead of concatenation

Replace with `[sigfile1, sigfile2, *verified_binaries]`

(RUF005)